alpaca_trade_api/rest.py

```diff
@@ -360,23 +365,27 @@ def replace_order(
             stop_price: str = None,
             trail: str = None,
             time_in_force: str = None,
-            client_order_id: str = None
+            client_order_id: str = None,
+            notional: str = None,
     ) -> Order:
         """
         :param order_id:
-        :param qty: str of int
```

Comment:
It doesn't seem that the Orders API supports replacement with fractional quantities or notional amounts (see [gobroker/oms/orderapi.go#L210-213](https://github.com/alpacahq/gobroker/blob/765a9fdbc1c81fd7abe79b7c317afff8c2c75a45/oms/orderapi.go#L210)), so I'm gonna revert this.